test/integration/workspace/test_workspace.py

```diff
@@ -524,6 +538,30 @@ def root_conanfile(self):
         c.run("workspace install", assert_error=True)
         assert "ERROR: Conanfile in conanws.py shouldn't have 'requires'" in c.out
 
+    def test_install_partial(self):
+        # If we want to install only some part of the workspace
+        c = TestClient()
+        c.save({"dep/conanfile.py": GenConanfile()})
+        c.run("create dep --name=dep1 --version=0.1")
+        c.run("create dep --name=dep2 --version=0.1")
+        c.save({"conanws.yml": "",
+                "liba/conanfile.py": GenConanfile("liba", "0.1").with_requires("dep1/0.1"),
+                "libb/conanfile.py": GenConanfile("libb", "0.1").with_requires("liba/0.1"),
+                "libc/conanfile.py": GenConanfile("libc", "0.1").with_requires("libb/0.1",
+                                                                               "dep2/0.1")},
+               clean_first=True)
+        c.run("workspace add liba")
+        c.run("workspace add libb")
+        c.run("workspace add libc")
+        for arg in ("libb", "libb liba"):
+            c.run(f"workspace install {arg} -g CMakeDeps -of=build")
```

Comment:
Can you add a test to show that when no path is passed, all are installed/no filtering is done? (If I understood the code correctly, that is)

Reply:
There is already a full functional test that does that.